es/index.html

```diff
@@ -14,68 +14,57 @@
 </head>
 <body>
   <main>
-    <h1>No pidas permiso para preguntar, sólo pregunta.</h1>
+    <h1>No preguntes por preguntar, sólo pregunta</h1>
```

Comment:
There is a difference in the statement intention between "No pidas permiso para preguntar" (prev) and "No preguntes por preguntar" (proposed).

* No pidas permiso para preguntar = Do not ask for permission to write your question.
* No preguntes por preguntar = Do not make questions just for the sake of making questions.

I've taken a look at other translations and they share the current intention: it's not about asking questions, it is about asking for permission first. I don't understand how the new title improves things.